Tester/Framework/Assert.php

```diff
@@ -47,11 +47,11 @@ class Assert
 	 * Checks assertion. Values must be exactly the same.
 	 * @return void
 	 */
-	public static function same($expected, $actual)
+	public static function same($expected, $actual, $message = "")
```

Comment:
I can rename it but message is common name in this context.

I will change it to single quotes